app/columns/[id]/page.js

```diff
@@ -3,14 +3,47 @@
 import Head from "next/head";
 import Date from "../../../lib/date";
 import Image from "next/image";
-import { Render } from "@9gustin/react-notion-render";
+import { Render, withContentValidation } from "@9gustin/react-notion-render";
 import "@9gustin/react-notion-render/dist/index.css";
 import ReactLoading from "react-loading";
 import { useQueries } from "@tanstack/react-query";
+import { useRouter } from "next/navigation";
+
+function PostImage({ className, media }) {
```

Comment:
Refactor this as its own component because it is used in multiple places (app/news/[id]/page.js)

Reply:
So I should just make a PostImage component right?

Reply:
@oukoutdam Yes, create a  new component in the component folder